contracts/Archetype.sol

```diff
@@ -185,17 +188,29 @@ contract Archetype is ERC721A__Initializable, ERC721AUpgradeable, ERC721A__Ownab
 
   function batchMintTo(
     Auth calldata auth,
-    address[] calldata to,
-    uint256[] calldata quantity,
+    address[] calldata toList,
+    uint256[] calldata quantityList,
     address affiliate,
     bytes calldata signature
   ) public payable {
-    if(quantity.length != to.length) {
+    if(quantityList.length != toList.length) {
       revert InvalidConfig();
     }
-    for (uint256 i=0; i< to.length; i++){
-      mintTo(auth, quantity[i], to[i], affiliate, signature);
+    uint256 quantity = 0;
+    for (uint256 i=0; i< quantityList.length; i++){
```

Comment:
How costly is this loop is in gas terms? Just out of curiosity

Reply:
It costs 123,128 gas units to run that for 1000 addresses/quantities
(123 gas units per loop)

Which is fairly cheap.

its when you loop through storage writes/reads that things get expensive.